components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -238,7 +247,13 @@ tweaking their $case/namelist_scream.xml file.
     ${DIN_LOC_ROOT}/atm/scream/tables/vn_table_vals.dat8,
     ${DIN_LOC_ROOT}/atm/scream/tables/vm_table_vals.dat8,
     ${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30_20220428.nc,
-    ${DIN_LOC_ROOT}/atm/scream/init/map_ne30_to_ne4_mono_20220502.nc
+    ${DIN_LOC_ROOT}/atm/scream/init/map_ne30_to_ne4_mono_20220502.nc,
+    <!-- ${DIN_LOC_ROOT}/atm/scream/init/map_ne30np4_to_ne120np4_mono_20220502.nc -->
+    <!-- ${DIN_LOC_ROOT}/atm/scream/init/map_ne30np4_to_ne512np4_mono_20220506.nc -->
+    ${DIN_LOC_ROOT}/atm/scream/init/init_ne4np4.nc
```

Comment:
This will need a comma at the end, otherwise it throws an obscure error during the `case.submit` command that is hard to track.

Reply:
Are you sure? Current master does not have a comma after the last input file, and it seems to run just fine.

Reply:
This is the second-to-last entry.

Reply:
Ah, I cannot read.